pkg/apply/processor/errors.go

```diff
@@ -16,6 +16,20 @@ limitations under the License.
 
 package processor
 
-import "errors"
+import (
+	"errors"
+	"strings"
+)
+
+const (
+	RunGuestFailed = "RunGuestFailed"
+)
+
+func IsRunGuestFailed(err error) bool {
+	if err == nil {
+		return false
+	}
+	return strings.Contains(err.Error(), RunGuestFailed)
```

Comment:
Is it better to adjust strings.Contains to strings.HasPrefix ？

Reply:
> Is it better to adjust strings.Contains to strings.HasPrefix ？
Considering that this error is wraped elsewhere, the Contains is more general.